frontend/src/features/housingCompany/HousingCompanyRealEstatesPage.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useState} from "react";
+
+import {Fieldset, IconCrossCircle} from "hds-react";
+import {useParams} from "react-router-dom";
+import {useImmer} from "use-immer";
+
+import {
+    useCreateRealEstateMutation,
+    useGetHousingCompanyDetailQuery,
+    useRemoveRealEstateMutation,
+} from "../../app/services";
+import {
+    ConfirmDialogModal,
+    FormInputField,
+    NavigateBackButton,
+    SaveButton,
+    SaveDialogModal,
+} from "../../common/components";
+import {IRealEstate} from "../../common/models";
+import {hitasToast} from "../../common/utils";
+
+const HousingCompanyRealEstatesPage = (): JSX.Element => {
+    const params = useParams();
+    const [isConfirmModalVisible, setIsConfirmModalVisible] = useState(false);
+    const [isEndModalVisible, setIsEndModalVisible] = useState(false);
+    const [idToRemove, setIdToRemove] = useState<string | null>();
```

Comment:
For slightly better clarity and consistency, this could be `realEstate(Id?)ToRemove`